02_classify_mutations/run_mutation_classification.py

```diff
@@ -212,6 +216,7 @@ def process_args():
                     gene,
                     classification,
                     gene_dir,
+                    filter_cancer_types=(not model_options.filter_all_data),
```

Comment:
This notation is weird to me. Why not just have a parameter called --filter_by_cancer_types?

Reply:
I wanted the default to be what we were doing before (and will likely continue to do after this), so I didn't want to add a parameter to these scripts that I'd then have to go back and add to all my bash/cluster scripts, etc. Having a flag to turn on the "new" filtering version made more sense at the script level.

I guess I could have the function also take the `filter_all_data` argument, which would make this logical not/negation unnecessary. Maybe that would make more sense.